subprojects/native/src/main/java/org/gradle/internal/nativeintegration/filesystem/services/NativePlatformBackedStat.java

```diff
@@ -30,6 +33,11 @@ public NativePlatformBackedStat(PosixFiles posixFiles) {
 
     @Override
     public int getUnixMode(File f) {
```

Comment:
❌ can you document the expected behavior in FileModeAccessor?

Can you also open an issue on https://github.com/gradle/native-platform to push this down into native-platform for 8.5?

Reply:
Yeah, last evening we decided to incorporate native-platform upgrade it in 8.4RC1, sorry for not updating this PR in time. See https://github.com/gradle/native-platform/pull/331

Reply:
The PR native platform milestone is released, I've switched to it